sal/class.json-api-site-base.php

```diff
@@ -419,6 +419,7 @@ function get_capabilities() {
 			'upload_files'        => current_user_can( 'upload_files' ),
 			'delete_users'        => current_user_can( 'delete_users' ),
 			'remove_users'        => current_user_can( 'remove_users' ),
+			'view_hosting'        => apply_filters( 'wpcom_json_api_site_can_view_hosting', false ),
```

Comment:
I think we should be using the `jetpack_` prefix, but I see that there are other [existing instances](https://github.com/Automattic/jetpack/blob/master/sal/class.json-api-site-base.php#L313) consistent with your approach – naming things is hard. Maybe someone on the JP side can give a definitive suggestion.

Reply:
`jetpack_` may indeed make more sense. More importantly, I would recommend adding a docblock so the parser can pick up that new filter and add it to the Codex. Something like this:

```suggestion
			/**
		 	 * Filter whether the Hosting section in Calypso should be available for site.
			 *
			 * @module json-api
			 *
			 * @since 8.2.0
			 *
			 * @param bool $view_hosting Can site access Hosting section. Default to false.
			 */
			'view_hosting'        => apply_filters( 'jetpack_json_api_site_can_view_hosting', false ),
```